Automattic/studio
# Switch "Local path" field to copy button instead of open new button

## Proposed Changes

Switches the "Local path" field to use a copy button instead of open new button.

**Before:**

![CleanShot 2024-09-10 at 03 48 40@2x](https://github.com/user-attachments/assets/3b49ba36-5899-43ba-854a-03974c721ba4)


**After:**

![CleanShot 2024-09-10 at 03 47 45@2x](https://github.com/user-attachments/assets/50b7bd2b-8b88-40f5-ac57-13d7856128bc)


## Testing Instructions

1. Open the "Settings" tab.
2. Verify clicking the copy icon adds the site's path to the clipboard.

## Pre-merge Checklist
